draft-api/src/main/scala/no/ndla/draftapi/service/ConverterService.scala

```diff
@@ -642,10 +642,12 @@ trait ConverterService {
         newContent <- cloneFilesForOtherLanguages(article.content, toMergeInto.content, isNewLanguage)
       } yield (newNotes, newContent)
 
-      val responsible = article.responsibleId match {
-        case Left(_)                    => None
-        case Right(Some(responsibleId)) => Some(Responsible(responsibleId, clock.now()))
-        case Right(None)                => toMergeInto.responsible
+      val responsible = (article.responsibleId, toMergeInto.responsible) match {
+        case (Left(_), _)                       => None
+        case (Right(Some(responsibleId)), None) => Some(Responsible(responsibleId, clock.now()))
+        case (Right(Some(responsibleId)), existing) if (existing.get.responsibleId != responsibleId) =>
+          Some(Responsible(responsibleId, clock.now()))
+        case (Right(_), existing) => existing
```

Comment:
Vil ikke _egentlig_ være noe problem siden du allerede fanger `None` casen, men syns dette er bittelitt penere og safere ved en eventuell refaktorering :smile: 
```suggestion
      val responsible = (article.responsibleId, toMergeInto.responsible) match {
        case (Left(_), _)                       => None
        case (Right(Some(responsibleId)), None) => Some(Responsible(responsibleId, clock.now()))
        case (Right(Some(responsibleId)), Some(existing)) if existing.responsibleId != responsibleId =>
          Some(Responsible(responsibleId, clock.now()))
        case (Right(_), existing) => existing
```

Reply:
Enig.

Reply:
Enig